src/services/command/command.ts

```diff
@@ -8,7 +8,7 @@ import { relativeToThisFile } from "../util/paths.js";
 import type { ArgsSpec } from "./arg.js";
 import type { Context } from "./context.js";
 
-export const AvailableCommands = ["sync", "list", "login", "logout", "whoami", "version"] as const;
+export const AvailableCommands = ["sync", "list", "login", "logout", "whoami", "version", "deploy"] as const;
```

Comment:
The order of these commands dictates where it shows up on the `README.md`. How do you feel about moving this closer to the top since it's such a cool command! 😎 

```suggestion
export const AvailableCommands = ["sync", "deploy", "list", "login", "logout", "whoami", "version"] as const;
```

Reply:
ahh ok, yup that makes sense 👍